index.html

```diff
@@ -40,6 +40,112 @@ <h1>I’m Soban. Glad to see you!</h1>
                 </div>
             </div>
         </div>
+        <div class="workspace">
```

Comment:
To ensure always that you are writing semantically correct markup, I recommend using a `section` tag as a wrapper for every section of your page instead of a `div` and I also recommend doing same for the about section too.

Reply:
Thanks almost forgot about it.